build/params_2k.go

```diff
@@ -120,3 +120,17 @@ const InteractivePoRepConfidence = 6
 const BootstrapPeerThreshold = 1
 
 var WhitelistedBlock = cid.Undef
+
+const DelegatedPoWFinality = 2
+const PoWFinality = 2
+const TendermintFinality = 1
+const MirFinality = 1
+const DummyFinality = 1
+const FilecoinECFinality = 2
```

Comment:
`2` may be too low for Filecoin EC. Let's put 5 for now (although it is also quite low). Eudico builds from `2k` so even on testnet we would have this value. Not a pressing problem, we can change it in the future.